dependency.lic

```diff
@@ -8,8 +8,9 @@ require 'base64'
 require 'yaml'
 require 'ostruct'
 require 'digest/sha1'
+require 'monitor'
```

Comment:
Not sure why I just rolled this from scratch instead of finding an existing solution.